### ISSUE TITLE
Huawei Android Oreo 8.0.0 - Emui 8.0.0 Beta4-test1 - App not working

### ISSUE THREAD

**Issue by gbento94** [rovo89/Xposed#375]
Hi,

I have the installed module FSBI 5.1.3 and trying to start it I receive the error "could not get response from Xposed framework". Module is activated and I rebooted several times already, without any  success. No clue what went wrong. The bug does not allow me to use this wonderful application.
[xposed_error_20180801_101036.log](https://github.com/rovo89/Xposed/files/2284817/xposed_error_20180801_101036.log)

**Comment by MW420BLAZEIT** [comment]
Can confirm
Xposed does install on EMUI 8
But modules dont work

**Comment by gbento94** [comment]
Yes, xposed works, but the module in question does not. (flat style bar indicator)

**Comment by mafamily2496** [comment]
I also install this version on emui8, some modules not work

**Comment by mafamily2496** [comment]
应用控制器 and amplify not work

**Comment by Keksebacker** [comment]
Are there any news?
I have successfuly installed the Beta4-test1 but all testet modules do not work.
Youtube Backrgound / Youtube AdAway / Flight mode helper


- Honor 8 FRD-L09 - EMUI 8
- Magisk root
- official Xposed Installer
